packages/fuel-gauge/src/policies.test.ts

```diff
@@ -379,16 +437,21 @@ describe('Policies', () => {
     });
 
     it('on ContractFactory when deploying contracts', async () => {
+      using launched = await launchTestNode();
+
+      const {
+        wallets: [wallet],
+      } = launched;
+
       const maxFee = 1;
 
-      const { binHexlified, abiContents } = getFuelGaugeForcProject(
-        FuelGaugeProjectsEnum.SCRIPT_MAIN_ARGS
+      const factory = new ContractFactory(
```

Comment:
This is weird, why is a script being passed into `ContractFactory`? I created #2787 to cover this and this test would fail then and it can be fixed at that point also.